### ISSUE TITLE
Piece Folders when choosing piece sets.

### ISSUE THREAD

**Issue by Eventlesstew** [WandererXII/lishogi#793]
A couple months ago, I made a mockup where all of Lishogi's many piece sets are organised into folders. (Below)

If more piece sets are planned to be added in the future, this would be a neat addition to allow more room for them and to reduce clutter.

I've also merged the "Chu Shogi" and "Kyoto Shogi" tabs into the "Variants" tab, where the folders on that tab correspond to the respective variant. This especially works for adding variants such as Tori Shogi or Okisaki Shogi.

Not sure if this is gonna get added, or at the very least, we get the "Variants" tab, but it's fun to bring up ideas.
![image](https://github.com/WandererXII/lishogi/assets/161836208/e716cb22-8a67-453d-925d-701fbdc0bec1)

**Comment by WandererXII** [comment]
Thanks for the idea, definitely worth considering, especially if more piece sets are added.
One advantage of showing all piece sets that comes to mind is that you don't have to click between multiple tabs to see them all, or switch between two piece sets to test them.

**Comment by Eventlesstew** [comment]
I agree, for normal pieces it should only be considered if there are too many pieces to show at once. You can also do page numbers or scrolling too, which would work better depending on how the Variant piece folders are implemented.

For variants though, the folders definitely work in their current state considering the folders are organised per variant. (Meaning that each folder has its own selection.)
